packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -411,6 +411,16 @@ describe('network stubbing', function () {
   })
 
   context('network handling', function () {
+    // @see https://github.com/cypress-io/cypress/issues/8497
+    it('can load transfer-encoding: chunked redirects', function () {
+      const url4 = 'http://localhost:3501/fixtures/generic.html'
+      const url3 = `http://localhost:3501/redirect?href=${encodeURIComponent(url4)}`
+      const url2 = `https://localhost:3502/redirect?chunked=1&href=${encodeURIComponent(url3)}`
+      const url1 = `https://localhost:3502/redirect?chunked=1&href=${encodeURIComponent(url2)}`
+
+      cy.visit(url1)
```

Comment:
should there be an assertion that url is url4?

Reply:
sure, why not, added